dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -1743,6 +1743,664 @@
+            "Milligram",
+            "Gallon",
+            "MetricTon",
+            "Ton",
+            "Pound",
+            "Ounce",
+            "Grain",
+            "PennyWeight",
+            "LongTonBritish",
+            "ShortTonUS",
+            "ShortHundredWeightUS",
+            "Stone",
+            "Dram"
+          ],
+          "x-ms-enum": {
+            "name": "WeightUnit",
+            "modelAsString": true
+          },
+          "description": "The weight Unit of measurement."
+        }
+      },
+      "required": [
+        "unit"
+      ]
+    },
+    "CurrencyResolution": {
+      "description": "Represents the currency entity resolution model.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseResolution"
+        },
+        {
+          "$ref": "#/definitions/QuantityResolution"
+        }
+      ],
+      "type": "object",
+      "x-ms-discriminator-value": "CurrencyResolution",
+      "properties": {
+        "iso4217": {
+          "type": "string",
+          "description": "The alphabetic code based on another ISO standard, ISO 3166, which lists the codes for country names. The first two letters of the ISO 4217 three-letter code are the same as the code for the country name, and, where possible, the third letter corresponds to the first letter of the currency name."
+        },
+        "value": {
+          "type": "number",
+          "format": "double",
+          "description": "The money amount captured in the extracted entity"
+        },
+        "unit": {
+          "type": "string",
+          "description": "The unit of the amount captured in the extracted entity"
+        }
+      },
+      "required": [
+        "value",
+        "unit"
+      ]
+    },
+    "BooleanResolution": {
+      "description": "A resolution for boolean expressions",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseResolution"
+        }
+      ],
+      "type": "object",
+      "x-ms-discriminator-value": "BooleanResolution",
+      "properties": {
+        "value": {
+          "type": "boolean"
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "DateTimeResolution": {
+      "description": "A resolution for datetime entity instances.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseResolution"
+        }
+      ],
+      "type": "object",
+      "x-ms-discriminator-value": "DateTimeResolution",
+      "properties": {
+        "timex": {
+          "$ref": "#/definitions/TimeExpression"
+        },
+        "dateTimeSubKind": {
+          "type": "string",
+          "enum": [
+            "Time",
+            "Date",
+            "DateTime",
+            "Duration",
+            "Set"
+          ],
+          "x-ms-enum": {
+            "name": "DateTimeSubKind",
+            "modelAsString": true
+          },
+          "description": "The DateTime SubKind"
+        },
+        "value": {
+          "type": "string",
```

Comment:
should we change add the 
 "format": "date-time"

See: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cognitiveservices/data-plane/Language/preview/2022-10-01-preview/common.json#L181

Reply:
@mssfang, I was thinking that the "date-time" format should be added to the "TimeExpression" definition instead. However, I haven't confirmed that the service actually returns the right format, which is supposed to be [RFC3339](https://www.rfc-editor.org/rfc/rfc3339#section-5.6) according to the [OpenAPI specification 2.0](https://swagger.io/specification/v2/). Do you know if the service returns this correctly?

Reply:
I think you are right. Btw, do you know what is an example of `Set` looks like?